posts/2020/07/Three-months-and-having-fun.md

```diff
@@ -0,0 +1,74 @@
+<!--
+.. title: Quansight Labs: what I learned in my first 3 months
+.. slug: A win-win all around
+.. date: 2020-07-03 01:00:00 UTC-05:00
+.. author: Matti Picus
+.. tags: Labs, NumPy, PyTorch, funding
+.. category:
+.. link:
+.. description:
+.. type: text
+-->
+
+I joined Quansight Labs at the beginning of April, splitting my time between
+PyTorch (as part of a group working under a contract) and contributing to the
+Python scientific data software stack, primarily to NumPy. I have found
```

Comment:
"Python scientific data software stack" sounds slightly odd. I'd use one of:
- Python scientific and data science software stack
- SciPy/PyData software stack